datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/ResolverUtils.java

```diff
@@ -135,15 +135,25 @@ public static Filter buildFilter(@Nullable List<FacetFilterInput> andFilters, @N
     public static Criterion criterionFromFilter(final FacetFilterInput filter) {
         Criterion result = new Criterion();
         result.setField(getFilterField(filter.getField()));
```

Comment:
Do we have tests for this method elsewhere? It will be good to test the new changes

Reply:
added